vizro-core/docs/pages/tutorials/first-dashboard.md

```diff
@@ -1,8 +1,8 @@
 # A first dashboard
 
-There is no setup needed for your first dashboard, thanks to the amazing [Py.Cafe](https://py.cafe/).
+There is no setup needed for your first dashboard, thanks to the amazing [PyCafe](https://py.cafe/).
 
-Click on the **Run and edit this code in Py.Cafe** link below to live-edit the dashboard.
+Click on the **Run and edit this code in PyCafe** link below to live-edit the dashboard.
```

Comment:
The link created by MkDocs plugin actually says "Py.Cafe" at present but I've asked the creators to accept a PR to change it to "PyCafe" so it is consistent with these changes to the docs.